tonic-build/Cargo.toml

```diff
@@ -4,15 +4,15 @@ categories = ["network-programming", "asynchronous"]
 description = """
 Codegen module of `tonic` gRPC implementation.
 """
-documentation = "https://docs.rs/tonic-build/0.12.3"
+documentation = "https://docs.rs/tonic-build/0.13.0-dev"
 edition = "2021"
 homepage = "https://github.com/hyperium/tonic"
 keywords = ["rpc", "grpc", "async", "codegen", "protobuf"]
 license = "MIT"
 name = "tonic-build"
 readme = "README.md"
 repository = "https://github.com/hyperium/tonic"
-version = "0.12.3"
+version = "0.13.0-dev"
```

Comment:
Can we just make the version `0.13.0` and not add the dev portion?

Reply:
Updated to `0.13.0`.